### PR TITLE
Update Inno Setup installer script

### PR DESCRIPTION
**Pull request by bovirus**
@CodingWonders 

- Program version get automatically by main exe file
- Add copyright info
- Copyright end year automatically get by current year
- Add icon for uninstaller in Windows contro Panel - Program installation
Etc.

### CODE REVIEW

**Comment by CodingWonders** [comment]
@bovirus, paths must not change since you're targeting a preview release

File: Installer/dt.iss [hunk 1]
```diff
@@ -65,33 +88,17 @@ Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{
...
-Source: ".\files\System.Runtime.CompilerServices.Unsafe.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
-Source: ".\files\System.Security*.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
 Source: ".\files\System.ServiceModel.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
 Source: ".\files\System.Windows.Forms.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
 Source: ".\files\System.Xml.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
 Source: ".\files\System.Xml.Linq.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
 Source: ".\files\WeifenLuo.WinFormsUI.Docking.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
 Source: ".\files\WeifenLuo.WinFormsUI.Docking.ThemeVS2012.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
-Source: ".\files\WindowsBase.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
-Source: ".\files\WindowsFormsIntegration.dll"; DestDir: "{#pfDir}"; Flags: ignoreversion
```
Comment (CodingWonders) [inline Installer/dt.iss]: Put these back

File: Installer/dt.iss [hunk 2]
```diff
@@ -110,70 +117,70 @@ Root: HKA; Subkey: "Software\Classes\Applications\{#MyAppExeName}\SupportedTypes
...
+
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; Flags: uninsdeletekey createvalueifdoesntexist
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; ValueType: dword; ValueName: "SkipQuestions"; ValueData: 1; Flags: uninsdeletevalue createvalueifdoesntexist
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; ValueType: dword; ValueName: "Pkg_CompleteInfo"; ValueData: 1; Flags: uninsdeletevalue createvalueifdoesntexist
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; ValueType: dword; ValueName: "Feat_CompleteInfo"; ValueData: 1; Flags: uninsdeletevalue createvalueifdoesntexist
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; ValueType: dword; ValueName: "AppX_CompleteInfo"; ValueData: 1; Flags: uninsdeletevalue createvalueifdoesntexist
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; ValueType: dword; ValueName: "Cap_CompleteInfo"; ValueData: 1; Flags: uninsdeletevalue createvalueifdoesntexist
+Root: HKCU; Subkey: "Software\DISMTools\Stable\InfoSaver"; ValueType: dword; ValueName: "Drv_CompleteInfo"; ValueData: 1; Flags: uninsdeletevalue createvalueifdoesntexist
```
Comment (CodingWonders) [inline Installer/dt.iss]: Replace `Stable` with `Preview`

File: Installer/dt.iss [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
+#define MyAppExeName       "DISMTools.exe"
+#define MyAppVersion       GetVersionNumbersString('.\files\' + MyAppExeName)
+#define MyAppPublisher     "CodingWonders Software"
+#define MyAppURL           "https://github.com/CodingWonders/DISMTools"
+#define MyAppAssocName     MyAppName + " project"
+#define MyAppAssocExt      ".dtproj"
+#define MyAppAssocKey      StringChange(MyAppAssocName, " ", "") + MyAppAssocExt
 
-#define pfDir "{commonpf}\DISMTools\Preview"
+#define pfDir               "{commonpf}\DISMTools\Stable"
```
Comment (CodingWonders) [inline Installer/dt.iss]:
```suggestion
#define pfDir               "{commonpf}\DISMTools\Preview"
```

**Review by CodingWonders — CHANGES_REQUESTED** [review]

**Comment by bovirus** [comment]
@CodingWonders 

I'm so sorry but I diodn't check if Preview branch use a different iss file.
Please check now.

**Comment by CodingWonders** [comment]
Could you do the commit?

**Comment by bovirus** [comment]
@CodingWonders 

How can do the commit?
Sorry but I'm not so expert about it?

**Comment by CodingWonders** [comment]
**Push** your changes to your branch, and they will appear here. Also, have you updated your branch with the change I made (**pulled** the changes)? (do that before pushing your changes)

Note that the words in bold are part of the common terminology of Git

**Comment by bovirus** [comment]
@CodingWonders 

I forked your full repository on my profile.
Then I loaded the changes in my repository and created a PR.
If you need more task please consider to do it.

**Comment by CodingWonders** [comment]
You can still push changes, though. I'll handle merging this PR.

**Comment by bovirus** [comment]
@CodingWonders 

For me you can merge.

**Comment by CodingWonders** [comment]
I'll do the necessary edits and merge the PR

**Comment by bovirus** [comment]
@CodingWonders 

In my update the changes are only about 

- Pre-processor variables
- [Setup] section

**Review by CodingWonders — APPROVED** [review]

**Comment by bovirus** [comment]
@CodingWonders 

Thanks for commit.
Where can I get Italian language for GUI? Which file/path?
I didn't find info on repository home page.

**Comment by CodingWonders** [comment]
@bovirus, there are Italian translations already implemented. Go to Options -> Personalization.

This is completely different from this PR's purpose, though

**Comment by bovirus** [comment]
image

image

image

image

There are some language issues.
Many times use infinite verbs when in Italian we use third person."Impegna" should be "Applica" (Apply).
Etc.
How can edit language files?

**Comment by CodingWonders** [comment]
Translations are scattered across the Visual Basic source files. I'll apply your proposals myself if that's what you prefer.

Could you pass this to a new issue?

**Comment by bovirus** [comment]
@CodingWonders 

I will open a new issue for translation.
Thanks.

**Pull request merged**